tests/unit/components/form/HdRange.spec.js

```diff
@@ -0,0 +1,99 @@
+import { wrapperFactoryBuilder } from 'tests/unit/helpers';
+import HdRange from '@/components/form/HdRange.vue';
+
+describe('HdRange', () => {
+  let wrapper;
+  const initialValue = 25;
+  const maxValue = 100;
+  const minValue = 0;
+  const wrapperFactory = wrapperFactoryBuilder(HdRange, {
+    propsData: {
+      name: 'testRange',
+      minValue,
+      maxValue,
+      value: initialValue,
+    },
+  });
+
+  beforeEach(() => {
+    wrapper = wrapperFactory();
+  });
+
+  test('renders component', () => {
+    expect(wrapper.html()).toMatchSnapshot();
+  });
+
+  test('renders with preselected value', () => {
+    expect(parseFloat(wrapper.find('input').element.value)).toBe(initialValue);
+  });
```

Comment:
The second assertion could be inside the first test as you initialize the component with a value already.

Reply:
👍